tests/integration_tests/import_export_tests.py

```diff
@@ -542,9 +542,11 @@ def test_import_override_dashboard_slice_reset_ownership(self):
         self.assertEqual(imported_slc.owners, [gamma_user])
 
         # re-import with another user shouldn't change the permissions
-        g.user = admin_user
```

Comment:
@michael-s-molina I don't believe your logic is correct, i.e., the original logic was actually correct. The intention of the test (as far as I can tell) is to import a dashboard into Superset (as the `gamma` user) and then reimport the same dashboard (as the `admin` user). Given that the dashboard existed the creator and owner should not change.

I _presume_ the `changed_by` doesn't change because none of the dashboard attributes (metadata) changed, but that's simply a hunch.

Reply:
Let me know how you want to proceed here. I assumed the tests were similar because they have the exact names with the only difference being new vs override. Given that you wanted to skip the test, I would keep it as is.

Reply:
@michael-s-molina I wonder if we should leave the test as-is and just skip it. It's the legacy version of the importer as well and thus it's _likely_ unused. Maybe @betodealmeida would have some thoughts regarding this?

Reply:
Yeah, it's a common pattern, where admins will manage dashboards under source control and periodically sync them to Superset. In that case you want to preserve the original ownership.

But like John said, if this is for the v0 importer (I'm on my phone, hard to see) then I wouldn't worry about it too much, since a lot of the behavior there is not strictly defined.

Reply:
@john-bodley I reverted and skipped the test.